services/intersection-api/api/pom.xml

```
@@ -119,7 +129,7 @@
 		<dependency>
 			<groupId>usdot.jpo.ode</groupId>
 			<artifactId>jpo-geojsonconverter</artifactId>
-			<version>2.1.1</version>
+			<version>3.0.4</version>
```

suggestion: point to GJC fix
```suggestion
			<version>3.0.5</version>
```